src/main/resources/openapi/cases.yaml

```diff
@@ -237,6 +237,41 @@ paths:
               schema:
                 $ref: './problem.yaml'
 
+
+  /cases/{caseId}/hearings/{hearingId}/events:
+    get:
+      tags:
+        - Cases
+      summary: Allows the retrieval of events for a given caseId and hearingId
```

Comment:
Think we should think about adding a simple unique `operationId: getEvents` and have it the same name in the Controller?  I just find it easier to search for the text "Find in Files..." that way

Reply:
What is operationId: getEvents?